### ISSUE TITLE
RequireStatementInCodeBock

### ISSUE THREAD

**Issue by patwork** [rrdelaney/jadelint#13]
Why it is invalid? Example code from http://jade-lang.com/reference/conditionals/

``` jade
- var user = { description: 'foo bar baz' }
- var authorised = false
#user
    if user.description
        h2 Description
        p.description= user.description
    else if authorised
        h2 Description
        p.description.
            User has no description,
            why not add one...
    else
        h1 Description
        p.description User has no description
```

```
example.jade
  ✖  line 4   Require Statement in Code Block
  ✖  line 7   Require Statement in Code Block
  ✖  line 12  Require Statement in Code Block

✖  3 warnings
```

BTW, these are warnings or errors?

edit: nevermind, dot after p.description. is actually working

**Comment by rrdelaney** [comment]
The "warnings" for errors was a typo I fixed a few hours ago. I didn't think it merited a release, but it'll probably go out later today. As far as the false positives, I'm investigating now

Edit: I found the culprit, should release a fix soon

**Comment by rrdelaney** [comment]
Rule is fixed in v0.2.13, thanks for the bug report!
